shell/browser/api/electron_api_web_request.cc

```diff
@@ -156,12 +157,19 @@ void ToDictionary(gin::Dictionary* details, extensions::WebRequestInfo* info) {
                  HttpResponseHeadersToV8(info->response_headers.get()));
   }
 
-  auto* web_contents = content::WebContents::FromRenderFrameHost(
-      content::RenderFrameHost::FromID(info->render_process_id,
-                                       info->frame_id));
-  auto* api_web_contents = WebContents::From(web_contents);
-  if (api_web_contents)
-    details->Set("webContentsId", api_web_contents->ID());
+  auto* render_frame_host =
+      content::RenderFrameHost::FromID(info->render_process_id, info->frame_id);
+  if (render_frame_host) {
+    details->Set("frame",
+                 WebFrameMain::From(details->isolate(), render_frame_host));
```

Comment:
I believe including `shell/common/gin_converters/frame_converter.h` will remove the need to manually convert the RenderFrameHost to a WebFrameMain here.

```suggestion
    details->Set("frame", render_frame_host);
```